languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -29148,6 +29212,40 @@ though, and has since been slightly modified)-->
       <example correction="casier|remise|débarras|armoire extérieure"><marker>locker</marker></example>
       <example><marker>remise</marker></example>
     </rule>
+      <rule id="VISIO" name="raccourci visio">
+          <antipattern>
+              <token>visio</token>
+              <token>raconte</token>
+              <token>comment</token>
+          </antipattern>
+          <antipattern>
+              <token>visio</token>
+              <token regexp="yes">Wettini|Rotchari|Baronti|Tnugdali|Edwardi|Fursei|Pauli|Sancti|Bernoldi|Karoli|in|:|de</token>
+          </antipattern>
+          <pattern>
+              <token regexp="yes">en|la|une|des|les|cette|ces</token>
+              <token>visio</token>
+          </pattern>
+          <message>« \1 » est un raccourci familier. Employez <suggestion>visioconférence</suggestion> dans un contexte plus sérieux.</message>
+          <example correction="visioconférence">Nous pourrons régler ce problème dans notre prochain entretien <marker>en visio</marker>.</example>
+          <example>Le but de la Visio Karoli Grossi est clairement de légitimer la succession des Carolingiens.</example>
+          <example>La Visio raconte comment le fier chevalier Tondale...</example>
+      </rule>
+      <rule id="FORM" name="form">
+          <pattern>
+              <token regexp="yes">forms?</token>
+          </pattern>
+          <message>« \1 » est un anglicisme. Employez <suggestion><match no="1" regexp_match="(form)(s)?" regexp_replace="questionnaire$2"/></suggestion>, <suggestion><match no="1" regexp_match="(form)(s)?" regexp_replace="formulaire$2"/></suggestion>, <suggestion><match no="1" regexp_match="(form)(s)?" regexp_replace="forme$2"/></suggestion>, <suggestion>mouler</suggestion>.</message>
+          <example correction="questionnaire|formulaire|forme|mouler">Je lui ai donné cette <marker>form</marker> a remplir</example>
+      </rule>
+      <rule id="TEAM" name="team">
+          <pattern>
+              <token regexp="yes">teams?</token>
```

Comment:
"Teams" is the name of a popular Microsoft product ("Microsoft Teams"). I would recommend to not include the plural form because of that.

Reply:
Understood. I'll change that, then.